### PR TITLE
Bug 1364045 - Convert the groups unit tests to Jest

### PR DESCRIPTION
**Pull request by camd**
--WIP:
Trying to get these tests working with Jest is hitting an error with ``taskcluster-client-web``.  Talked to Dustin, and it appears to be an issue with the version of ``hawk`` that's getting installed with their package.  

Filed upstream bug https://bugzilla.mozilla.org/show_bug.cgi?id=1523376 to hopefully address this.

### CODE REVIEW

**Comment by edmorley** [comment]
A possible workaround in the meantime would be to use the `resolve` property in `package.json` to force an older point release of v7.x to be installed:
https://github.com/mozilla/treeherder/blob/dd3e0f33a1763c3e5f8ca64d740124c1343874a4/package.json#L13-L15

**Comment by edmorley** [comment]
I think I've figured out what's causing this - I've left a comment on the upstream issue:
https://bugzilla.mozilla.org/show_bug.cgi?id=1523376#c3

For now this can be worked around by adding `browser: true` to the Jest preset options in `.neutrinorc.js` (and the Hawk yarn resolutions pinning can be removed).

**Comment by camd** [comment]
> I think I've figured out what's causing this - I've left a comment on the upstream issue:
> https://bugzilla.mozilla.org/show_bug.cgi?id=1523376#c3
> 
> For now this can be worked around by adding `browser: true` to the Jest preset options in `.neutrinorc.js` (and the Hawk yarn resolutions pinning can be removed).

Man, I'm going to miss you.  :)  Nice sleuthing on this!  I had to fix Bug 1504713 as well, to get this to work.  But that was simple enough.  Thanks!!!

File: .neutrinorc.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ module.exports = {
       '@neutrinojs/jest',
       {
         setupFilesAfterEnv: ['<rootDir>/tests/ui/unit/test-setup.js'],
+        browser: true,
```
Comment (edmorley) [inline .neutrinorc.js]: Could you add a comment linking to the explanation in the upstream bug?

**Pull request merged**